themes/gatsby-theme-blog/src/components/post-footer-content.js

```diff
@@ -0,0 +1,42 @@
+import React, { Fragment } from "react"
+import { Link } from "gatsby"
+import { Styled, Flex } from "theme-ui"
+
+import Bio from "../components/bio"
+
+/**
+ * Shadow me to add your own footer content
+ */
+
+export default ({ previous, next }) => (
+  <Fragment>
+    <Styled.hr />
+    <Bio />
+    {(previous || next) && (
```

Comment:
Is this part new?

Reply:
the check is new, but not the next/previous pagination.